tests/PHPFluent/EventManager/Listener/PThreadTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace PHPFluent\EventManager\Listener;
+
+use PHPFluent\EventManager\Event;
+
+/**
+* @covers PHPFluent\EventManager\Listener\PThread
+*/
+class PThreadTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        parent::setUp();
+        if (! extension_loaded('pthreads')) {
+            $this->markTestSkipped('The pthreads extension is not available.');
+        }
```

Comment:
I'm sorry, I didn't saw that. Much better than put on `require-dev`, since it's a PHP extension